scripts/semantic-commits/parse-changelog.js

```diff
@@ -2,12 +2,7 @@
 const fs = require('fs')
```

Comment:
Would these changes be useful in `develop`? Maybe they could be cherry-picked into another PR against `develop`?

Reply:
@emilyrohrbough , thoughts?

Reply:
I have a branch in progress for this.